### DISCUSSION TITLE
BSP Tiling Mode

### DISCUSSION THREAD

**Discussion by codebedouin** [nikitabobko/AeroSpace#903] [feature-ideas]
I know that Aerospace is an i3-inspired window manager, but as someone who (happily!) switched over from Yabai + SKHD, BSP is the only feature I really miss. It makes efficient use of screen real estate for those of us that may have multiple windows open on a single workspace.

**Comment by nikitabobko** [comment]
#260

**Comment by pszypowicz** [comment]
FYI #2076 - have a fork in progress that may touch this corner. Brew tap: pszypowicz/tap/aerospace-bsp
